API/helpers/populate.js

```diff
@@ -12,7 +12,15 @@ exampleDocuments = [
 
 mongoose.connect('mongodb://localhost/grapp-dev');
 
-exampleDocuments.map(data => {
+const promises = exampleDocuments.map(data => {
   const ex = new Examplemodel(data);
-  ex.save();
+  console.log(`Creating ${data.name}...`);
+  return ex.save().then(ex => {
+  	console.log(`Successfully saved ${ex.name}`)
```

Comment:
I like the dialogue, but I think it might be a bit verbose - maybe we just need `Saved ${ex.name}`  and not the 'creating ...'. If we want to alert users to an error, let's throw a `.catch` in. Just thinking once we start populating other data (if we're adding dummy channels, posts, etc) we probably don't need two lines per document :stuck_out_tongue_closed_eyes: